app/assets/scripts/main.js

```diff
@@ -65,7 +66,10 @@ render((
         <Route path='editor/*' component={Editor} pageClass='editor' />
         <Route path='explore' component={Explore} pageClass='explore' />
         <Route path='analytics' component={Analytics}>
-          <IndexRoute component={AnalyticsIndex} pageClass='analytics' />
+          <Route path='road' component={Analytics} >
+            <Route path=':vpromm' component={AAFieldMap} pageClass='analytics-aa'/>
+          </Route>
+          <Route path='main' component={AnalyticsIndex} pageClass='analytics' />
```

Comment:
You should be using a IndexRoute here. Looks better than /main.
Any reason you went with this approach?